src/pages/[latitude]/[longitude]/[zoom]/index.tsx

```diff
@@ -114,19 +102,20 @@ export const MainPage = () => {
           error={error}
           downloadData={downloadData}
         />
-        <MainMap
-          showZoomModal={showZoomModal}
-          setShowZoomModal={setShowZoomModal}
-          showInfoModal={showInfoModal}
-          setShowInfoModal={setShowInfoModal}
-          loading={loading}
-          parkingLots={parkingLots}
-          savedBounds={savedBounds}
-          setBounds={setBounds}
-          viewport={viewport}
-          setViewport={setViewport}
-          mapRef={mapRef}
-        />
+        {initialized && (
```

Comment:
Wait until initialized so we can just update the starting viewport rather than moving the position after the map has loaded